index/generator/library/library.go

```diff
@@ -202,6 +216,27 @@
 	return nil
 }
 
+func iconExists(iconUrl string) bool {
+	resp, err := http.Get(iconUrl)
```

Comment:
IMO I think this can be dismissed. The value comes directly from the stacks/samples and this format of `http.Get` is used also in other cases (e.g the [encodeToBase64()](https://github.com/devfile/registry-support/blob/main/index/server/pkg/util/util.go#L81) func )

Reply:
This might fix this security alert:
```suggestion
        /* #nosec G107 -- iconUrl is taken from the index file.  Stacks / Samples with URLs to a devfile icon should be vetted beforehand */
	resp, err := http.Get(iconUrl)
```

Same as how this was handled in [encodeToBase64()](https://github.com/devfile/registry-support/blob/4bc237b077e1bd478ea24195fac7c35fd2975441/index/server/pkg/util/util.go#L90).